compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/OnClick.skiko.kt

```diff
@@ -122,6 +124,7 @@ fun Modifier.onClick(
             val on2xClickState = rememberUpdatedState(onDoubleClick)
             val onLongClickState = rememberUpdatedState(onLongClick)
             val keyboardModifiersState = rememberUpdatedState(keyboardModifiers)
+            val focusRequester = remember { FocusRequester() }
```

Comment:
What happen if the user will add two `onClick` modifiers - one for the left click, and one for the right click? Will the focus be requested for the left click?

Reply:
I'll check. My assumption is that focus will be requested for the left click and for the right click as well.

Reply:
yes, the component gets focus (if focusable is added) for both left and right clicks.

It's what expected in my opinion. Do you think so too?

Reply:
Yes, onClick should behave the same for all buttons